components/DynamicContent/src/index.scss

```diff
@@ -32,7 +32,6 @@
 
   display: flex;
   flex-direction: column;
-  gap: var(--denhaag-dynamic-content-card-gap);
```

Comment:
Is deze token nog ergens anders gebruikt? De `--denhaag-dynamic-content-card-gap`, zo niet dan mag deze ook uit de dynamic content card tokens verwijderd worden.
Anders zit er een ongebruikte token.

Reply:
Ik kon hem via de search nergens anders vinden, maar vond het een beetje link om hem zomaar weg te halen, weet niet GDH hierin als best practice heeft. Zover ik kan zien wordt hij nergens anders gebruikt in ieder geval.. gewoon weghalen dan maar?

Reply:
laat de token maar, np